app/controllers/DataSetController.scala

```diff
@@ -220,6 +222,9 @@ class DataSetController @Inject()(userService: UserService,
           js <- dataSetService.publicWrites(dataSet, request.identity, organization, dataStore)
           _ = request.identity.map { user =>
             analyticsService.track(OpenDatasetEvent(user, dataSet))
+            if (dataSet.isPublic && dataSet._organization != user._organization) {
```

Comment:
@normanrz @Albane-dlv is this the condition we want here? My understanding of the tag is that the user has viewed a dataset that is public and that is of a different organization.
Alternatively we could look at wether it is linked to a publication. What do you think?

Reply:
I was thinking of: user has viewed a public dataset from any organisation. If that is possible, it would be great!

Reply:
Sure, I changed it now :)